pom.xml

```diff
@@ -50,14 +50,29 @@
   <distributionManagement>
     <repository>
       <id>sonatype-ossrh</id>
-      <url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
+      <url>http://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
     </repository>
     <snapshotRepository>
       <id>sonatype-ossrh</id>
-      <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+      <url>http://oss.sonatype.org/content/repositories/snapshots</url>
     </snapshotRepository>
   </distributionManagement>
 
+  <!-- needed for JDK 1.6 which doesn't support TLS 1.2 -->
```

Comment:
Ouch! Really? Maybe it's time to forget about JDK 1.6 for this project anyway. (if we want, we can still have `target` set to 1.6, but use 1.7+ to build?)

Reply:
I've been able to setup JDK 1.7 instead, and get rid of these POM changes

Reply:
I've been able to setup JDK 1.7 instead, and get rid of these POM changes